FATN_to_USB/tests/test_display_detector.py

```
@@ -0,0 +1,505 @@
+    def test_detect_framebuffer_not_available(self):
+        """Test framebuffer detection when not available."""
+        with patch('os.path.exists', return_value=False):
+            result = detect_framebuffer()
+
+        assert result is False
+
+
+class TestDisplayInitialization:
+    """Tests for display initialization."""
+
+    def test_try_init_st7789_success(self, mock_st7789_display):
```

Ruff (0.14.14)

Unused method argument: `mock_st7789_display`

(ARG002)